
package Heart.proto;

message ServiceRegister {
	optional string seviceName = 1;
}

message ServiceUnregister {
	optional string seviceName = 1;
}

message ConsoleLogUpdate {
	optional string logUpdate = 1;
}
